cmd/kubelet/app/server.go

```diff
@@ -355,6 +255,24 @@ func makeEventRecorder(s *componentconfig.KubeletConfiguration, kubeDeps *kubele
 	}
 }
 
+// Returns the most likely identifer for the current Node
+func curNodeIdentifier(s *options.KubeletServer, kd *kubelet.KubeletDeps) (string, error) {
```

Comment:
I ended up removing the whole function, turned out to be a duplicate of `getNodeName`